components/backstage/index.js

```diff
@@ -23,6 +25,16 @@ function mapStateToProps(state) {
     const enableCommands = config.EnableCommands === 'true';
     const enableOAuthServiceProvider = config.EnableOAuthServiceProvider === 'true';
 
+    let canCreateCustomEmoji = haveISystemPermission(state, {permission: Permissions.MANAGE_EMOJIS});
+    if (!canCreateCustomEmoji) {
```

Comment:
I think I saw a discussion about this, but to clarify, is it the case that if I have team-level permission to create custom emojis, I can create custom emojis anywhere, since custom emojis are system wide not team wide?

Reply:
Yes that's the unfortunate case

Reply:
Yes, It's exactly that. This is the quick solution to migrate emoji permissions to the permissions system. But the plan is to include a nullable "TeamId" field in the Custom emojis, so we'll be able to allow users to have team custom emojis and system custom emojis, and the manage_emojis permissions will behave like the other permissions.